cmd/cri-containerd/options/options.go

```diff
@@ -18,38 +18,56 @@ package options
 
 import (
 	"flag"
+	"os"
+	"time"
 
+	"github.com/BurntSushi/toml"
 	"github.com/containerd/containerd"
 	"github.com/spf13/pflag"
 )
 
-// CRIContainerdOptions contains cri-containerd command line options.
-type CRIContainerdOptions struct {
+const (
+	// "default path to the config file"
+	configFilePathArgName = "config-file-path"
+)
+
+//Config contains cri-containerd toml config
```

Comment:
nit: Can we start your comment with a space, and also for some following comments? :)